README.md

```diff
@@ -60,8 +60,10 @@ The wrapper script depends on `bcftools` and `parallel` (GNU Parallel).
 
 ## Mapping process
 For each variant:
-* Query VEP with default distance to the gene (5,000 bases either way). Consider only consequences which affect **either a protein coding transcript or a miRNA**. Find the most severe consequence for either of those transcripts, according to the list of priorities described [on Ensembl website](https://www.ensembl.org/info/genome/variation/prediction/predicted_data.html). Output **all non-redundant** consequences of this (most severe) type.
-* If we found nothing during the previous step, repeat VEP search for this variant, now with a distance up to 500,000 bases either way. If there are any consequences which affect a **protein coding** transcript, choose the most severe consequence type (usually this will be either an upstream or a downstream gene variant) and output **a single consequence with the smallest distance**.
+1. Query VEP with default distance to the gene (5,000 bases either way). Consider only consequences which affect **either a protein coding transcript or a miRNA**. Find the most severe consequence for either of those transcripts, according to the list of priorities described [on Ensembl website](https://www.ensembl.org/info/genome/variation/prediction/predicted_data.html). Output **all non-redundant** consequences of this (most severe) type.
+2. *Optionally:* if we found nothing during the previous step, repeat VEP search for this variant, now with a distance up to 500,000 bases either way. If there are any consequences which affect a **protein coding** transcript, choose the most severe consequence type (usually this will be either an upstream or a downstream gene variant) and output **a single consequence with the smallest distance**.
+
+Step 2 can be enabled with the `--enable-distant-querying` flag to the core pipeline. It is disabled by default, as well as in the [run_consequence_mapping.sh](/consequence_mapping/run_consequence_mapping.sh) wrapper script and in the ClinVar batch processing use case. 
```

Comment:
This sentence is a bit unclear. Suggest re-phrasing to "Distant querying, while disabled by default, can be enabled in the wrapper script as well as the ClinVar batch processing scripts by adding --enable-distant-querying flag to the core pipeline."

Reply:
Thank you, rephrased in https://github.com/EBIvariation/vep-mapping-pipeline/pull/1/commits/742307d78921ca069d482303ac63273fb7a3a20a